scalajvm/public/stylesheets/main.css

```diff
@@ -94,24 +96,16 @@ ul.menu {
 	background: #445566;
 }
 
-.quote-time {
-    float: right;
-}
-
-.clear {
-    clear: both;
-}
```

Comment:
You have removed the `clear` class, but it's still used in a `quote.scala.html`. Either replace it there too or put it back.